src/if_py_both.h

```diff
@@ -7325,12 +7325,15 @@ populate_module(PyObject *m)
 	return -1;
     }
 
+# if PY_VERSION_HEX < 0x30d00f0
```

Comment:
0x30d00f0 means 3.13.0 final, but the reported version is beta1.
Should be 0x30d00b0 (or 0x30d00a0 or 0x30d0000)?

Reply:
I changed it to 3.12.0a7+